uni/assets/text/TermsAndConditions.md

```diff
@@ -4,50 +4,144 @@
+These terms and conditions ("Terms", "Agreement") are an agreement between NIAEFEUP ("us", "we" or "
+our"), Núcleo de Informática da Associação de Estudantes da Universidade do Porto, and you ("
+User", "you" or "your"). This Agreement sets forth the general terms and conditions of your use of
+the Uni mobile application and any of its products or services (collectively, "Mobile Application"
+or "Services").
 \
 \
 **Accounts and membership**
 
-To use the Mobile Application, you are required to hold a digital account provided by Universidade do Porto. We are in no way affiliated with this university. Any act performed by you through the Mobile Application is subject to the [Universidade do Porto's terms and conditions](https://sigarra.up.pt/up/pt/web_base.gera_pagina?p_pagina=termos%20e%20condicoes).
-By using your faculty account in the Mobile Application, you are responsible for maintaining the security of your account and you are fully responsible for all activities that occur under the account and any other actions taken in connection with it. We will not be liable for any acts or omissions by you, including any damages of any kind incurred as a result of such acts or omissions.
-Any interaction with your data from Universidade do Porto's platform is made directly through it, and is not collected by us in any way.
-If you choose to select the option to keep you logged in, you understand that your sign-in details may be stored locally, but not in any way shared with us. These details are stored in the application data and we hold no responsibility for ensuring their concealment.
+To use the Uni Mobile Application, you are required to hold a digital account provided by
+Universidade
+do Porto. We are in no way affiliated with this university. Any act performed by you through the
+Mobile Application is subject to
+the [Universidade do Porto's terms and conditions](https://sigarra.up.pt/up/pt/web_base.gera_pagina?p_pagina=termos%20e%20condicoes).
+By using your faculty account in the Mobile Application, you are responsible for maintaining the
+security of your account and you are fully responsible for all activities that occur under the
+account and any other actions taken in connection with it. We will not be liable for any acts or
+omissions by you, including any damages of any kind incurred as a result of such acts or omissions.
+Any interaction with your data from Universidade do Porto's platform is made directly through it,
+and is not collected by us in any way.
+If you choose to select the option to keep you logged in, you understand that your sign-in details
+may be stored locally, but not in any way shared with us. These details are stored in the
+application data and we hold no responsibility for ensuring their concealment.
+\
+\
+**Ownership of Content:**
```

Comment:
I would suggest removing the colon in the new topics, to make it consistent with the old topics.